ci/amazon.dockerfile

```diff
@@ -6,15 +6,20 @@ RUN \
     yum clean all \
     && yum upgrade -y \
     && yum groupinstall -y "Development Tools" \
+    && yum install -y epel-release \
+    && yum-config-manager --enable epel \
     && yum install -y \
            wget \
            cmake \
            pkgconfig \
            openssl-devel \
-           sqlite-devel
+           sqlite-devel \
+           libsodium-devel \
```

Comment:
Seems we unnecessary install libsodium-devel as we compile it from sources later